package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "http-routing",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "lint": "eslint .",
+    "pretest": "npm run lint"
```

```suggestion
    "pretest": "npm run lint",
    "start": "nodemon server.js"
```
Install nodemon (```npm i nodemon -D```) and include a start script like above to make testing server easier.